pkg/alertmanager/alertmanager_template.go

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+
+package alertmanager
+
+import (
+	"encoding/json"
+	tmpltext "text/template"
+
+	tmplhtml "html/template"
+	"net/url"
```

Comment:
nit: those don't seem properly formatted

Reply:
Right, should be fixed in https://github.com/grafana/mimir/pull/3849/commits/22b0c7809e0e5907aa5902a2eb22d57ec0637ac2